src/plonemeeting/portal/core/sync_utils.py

```
@@ -26,14 +25,14 @@
 
 def _call_delib_rest_api(url, institution):  # pragma: no cover
     start_time = time.time()
-    logger.info("REST API CALL TO {0}".format(url))
+    logger.info(f"REST API CALL TO {url}")
     response = requests.get(
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)